grammar.js

```diff
@@ -603,7 +612,7 @@ module.exports = grammar({
           $.identifier,
       ),
       '=>',
-      $.expression,
+      optional($._block),
```

Comment:
this is the one of two meaningful changes in this PR, the rest is just to make it compile